index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.8/css/line.css"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Community Pharmacists Conference</title>
+  </head>
+  <body>
+    <!------------------------------HEADER START---------------------------------->
+    <header class="head">
+      <nav class="navbar">
+        <div class="up">
+          <ul class="holder">
+            <li>
+              <a href="#"> <i class="uil uil-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"> <i class="uil uil-twitter"></i></a>
+            </li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Log out</a></li>
+          </ul>
+        </div>
+        <div class="down">
+          <div class="holder">
+            <a href="index.html" class="nav-logo">ACPN 2023</a>
+            <ul class="nav-menu">
+              <li class="nav-list">
+                <a href="index.html" class="nav-link">Home</a>
+              </li>
+              <li class="nav-list">
+                <a href="about.html" class="nav-link">About</a>
+              </li>
```

Comment:
- You have done well with your first project so far, but to help with the UI/UX of your site, I suggest that you do not have an `home` link in the home page of your site. Please fix this 🙏

Reply:
i don't understand this statement @mckent05